specification/apimanagement/resource-manager/Microsoft.ApiManagement/2017-03-01/apimapis.json

```diff
@@ -2039,6 +2477,17 @@
       "pattern": "^[^*#&+:<>?]+$",
       "x-ms-parameter-location": "method"
     },
+    "ApiIdRevParameter": {
+      "name": "apiId",
```

Comment:
>apiId [](start = 15, length = 5)

nit: should we rename to apirevId

Reply:
I just cant comprehend, how user will generate this id in the sdk... I guess maybe we need to provide a utility function.

---
In reply to: [159551481](https://github.com/darrelmiller/azure-rest-api-specs/pull/1#discussion_r159551481) [](ancestors = 159551481)